src/installer/cdk/src/index.ts

```diff
@@ -244,6 +250,7 @@ async function main() {
         },
       },
     },
+    cache: codebuild.Cache.local(codebuild.LocalCacheMode.DOCKER_LAYER, codebuild.LocalCacheMode.CUSTOM),
```

Comment:
I would prefer we keep our PR's focused (I know, sometimes we don't do a good job at this ourselves):
- 1st PR for cidr2 enhancements
- 2nd PR for generic improvements (caching, buildimage, and container updates)

Reply:
yup this is the idea, this is leaked since to validate the PR i am stuck with the slow ci. It takes minimum 2-3 hour to test a single change, getting very frustrating now.  The CI alone take 1/2 hour to build in codebuild and step functions another 1/2 hour. Was trying to fix this but no luck.

The local deploy functionality is kind of broken or not well documented, since the phase deploy don't update the nested stack. Outputs need to be copied manually from dynamodb for every phases. At this state this is in the shape that my team is leaning towards moving back to terraform+terragrunt :(.

Reply:
It's currently 2:30 am here - let me see if I can't get someone to provide you with better guidance tomorrow.